ui/client/components/graph/Graph.js

```diff
@@ -1,22 +1,21 @@
-import * as dagre from "dagre"
+/* eslint-disable i18next/no-literal-string */
 import * as joint from "jointjs"
 import "jointjs/dist/joint.css"
-import _, {cloneDeep} from "lodash"
+import _, {cloneDeep, defer} from "lodash"
 import PropTypes from "prop-types"
 import React from "react"
 import svgPanZoom from "svg-pan-zoom"
-import cssVariables from "../../stylesheets/_variables.styl"
 import {getProcessCategory, getSelectionState} from "../../reducers/selectors/graph"
 import {getLoggedUser, getProcessDefinitionData} from "../../reducers/selectors/settings"
 import "../../stylesheets/graph.styl"
 import "./svg-export/export.styl"
-import * as EspNode from "./EspNode"
 import * as GraphUtils from "./GraphUtils"
 import * as JointJsGraphUtils from "./JointJsGraphUtils"
 import EdgeDetailsModal from "./node-modal/EdgeDetailsModal"
 import NodeDetailsModal from "./node-modal/NodeDetailsModal"
 import NodeUtils from "./NodeUtils"
 import {prepareSvg} from "./svg-export/prepareSvg"
+import {drawGraph, directedLayout, isBackgroundObject, createPaper} from "./GraphPartialsInTS"
```

Comment:
I know this is an unusual solution, but I didn't want to rewrite (yet) the entire Graph module to ts.